test/required_with_ref.js

```diff
@@ -7,6 +7,7 @@ var util = require('util')
 var fs = require('fs')
 var assert = require('chai').assert
 var Validator = require('../lib/validator')
+var expect = require('chai').expect;
```

Comment:
When making changes in other's projects, we should stick with the tools they've already chosen, so `assert` instead of `expect`